File: components/permissions/src/test/java/org/cloudfoundry/credhub/services/TestOAuth2IssuerService.java
package org.cloudfoundry.credhub.services;

import org.springframework.context.annotation.Profile;
import org.springframework.stereotype.Service;

import org.cloudfoundry.credhub.auth.OAuth2IssuerService;

@Service
@Profile("unit-test")
public class TestOAuth2IssuerService implements OAuth2IssuerService {
  @Override
  public String getIssuer() {
    return "https://example.com:8443/oauth/token";
  }
}

File: components/permissions/src/test/kotlin/org/cloudfoundry/credhub/services/TestOAuth2IssuerService.kt
package org.cloudfoundry.credhub.services

import org.cloudfoundry.credhub.auth.OAuth2IssuerService
import org.springframework.context.annotation.Profile
import org.springframework.stereotype.Service

@Service
@Profile("unit-test")
class TestOAuth2IssuerService : OAuth2IssuerService {
    override fun getIssuer(): String {
        return "https://example.com:8443/oauth/token"
    }
}
